rules/anti-static/base64/function_names.yara

```diff
@@ -122,8 +122,9 @@ rule base64_php_functions_multiple: critical {
     $f_wp_nonce_field     = "wp_nonce_field" base64
     $f_wp_verify_nonce    = "wp_verify_nonce" base64
 
-    $not_comment     = "// processing instruction, e.g. <?php ?>"
-    $not_mongosh_php = { 3C 3F 70 68 70 00 00 00 01 0C 51 61 03 00 00 00 02 00 00 00 3F 3E }
```

Comment:
This was a bit too specific and no longer applied to updated versions of `mongosh`.